### DISCUSSION TITLE
Renovate PR not being created and repo not showing up in dashboard

### DISCUSSION THREAD

**Discussion by JordanWeatherby** [renovatebot/renovate#29678] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

Hi, we have renovate working fine on two repos and I tried to add it to a third repo a few days ago, https://github.com/NetPurpose/public-website-cms

![image](https://github.com/renovatebot/renovate/assets/47519158/da762427-f0ca-48fb-80ff-a9a79986f444)

But it is not showing up in our https://developer.mend.io/github/NetPurpose dashboard and no PR has been made 

![image](https://github.com/renovatebot/renovate/assets/47519158/57c00284-69fe-4742-9b55-114a97126bd7)

Hope someone has any idea why this is happening

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by HonkingGoose** [comment]
Silly idea incoming! :upside_down_face: Maybe you can "force" Renovate to start working, by putting a Renovate config file in your repository?

Steps for silly idea:

1. Create a `renovate.json` file in your repository with this basic config in it:
    ```json
    {
      "extends": ["config:recommended"]
    }
    ```
2. Wait a bit for Renovate to _maybe_ pick up the repository, and create a Dependency Dashboard and update PRs.
3. Customize the configuration to your liking.
4. Enjoy working Renovate bot.

If my silly idea does not work, you'll have to wait for a maintainer to respond. :wink:

**Comment by JordanWeatherby** [comment, marked as answer]
Ah it actually finally made the PR last night, not sure why it took days to happen but at least it has finally happened!

**Reply by HonkingGoose** [reply]
Yay! :partying_face: You should use Renovate's onboarding PR workflow then, that's better than my silly idea! :wink: 

The [Renovate docs, known limitations](https://docs.renovatebot.com/known-limitations/) also say that it can take some time for Renovate to start working.
